File: java/iniciante/ex-1009.java
package application;

import java.util.Locale;
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		String nome;
		double salarioFixo, totalVendas, total;
		
		nome = sc.nextLine();
		salarioFixo = sc.nextDouble();
		totalVendas = sc.nextDouble();
		
		total = salarioFixo + totalVendas*0.15;
		
		System.out.printf("TOTAL = R$ %.2f", total);
		System.out.println();
		
		sc.close();

	}

}

File: c++/iniciante/ex1009.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(){
	
	string nome;
	double salarioFixo, vendasMes, total;
	
    cin >> nome;
    cin >> salarioFixo;
    cin >> vendasMes;
    
    total = salarioFixo + (vendasMes*0.15);
    
    cout << "TOTAL = R$ " << fixed << setprecision(2) << total << endl;
    
    system("pause");
    
    return 0;
}
